reconcile/test/test_terraform_cloudflare_resources.py

```diff
@@ -20,6 +21,18 @@ def query_data(external_resources):
         namespaces=[
             NamespaceV1(
                 name="namespace1",
+                clusterAdmin=True,
+                cluster=ClusterV1(
+                    name="test-cluster",
+                    serverUrl="http://localhost",
+                    insecureSkipTLSVerify=None,
+                    jumpHost=None,
+                    automationToken=None,
+                    clusterAdminAutomationToken=None,
+                    spec=None,
+                    internal=None,
+                    disable=None,
+                ),
```

Comment:
More of an fyi: Chris added a nice way to create generated classes from sparse dicts https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L102

Specifically designed to keep boilerplate in tests low. In this case it is not really important, but I thought it might be worth advertising Chris' awesome new feature :)